src/cmd/serial.js

```diff
@@ -919,8 +920,22 @@ module.exports = class SerialCommand {
 		return promise.finally(cleanUpFn);
 	}
 
+	/**
+	 * This is a wrapper function created so _serialWifiConfig can return the
+	 * true promise state, but the wrapper can still act like it handled any
+	 * failures gracefully as it acted before testing was attempted.
+	 */
```

Comment:
This comment is probably not valuable long term, I should probably remove it and/or reword it.